.github/actions/rust-build-release/tests/conftest.py

```
@@ -8,6 +8,7 @@
 import shutil
 import sys
 import typing as typ
+from dataclasses import dataclass  # noqa: ICN003
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `ICN003`)

Remove unused `noqa` directive

(RUF100)